investigate_user_input.py

```diff
@@ -0,0 +1,24 @@
+from constants import 
+
+class GetName():
```

Comment:
This file appears to contain exploratory code that is not relevant for the Bites game and not used. This file should be deleted.